content/basics/use-ably.textile

```diff
@@ -0,0 +1,123 @@
+
+The advantage to protocol adapters is that they require fewer resources and network overhead, however they do not support the full set of Ably features.
+
+We offer protocol adapters for:
+
+* "External MQTT libraries":#mqtt
+* "Ably SSE and EventSource API":#sse-eventsource
+* "Pusher and PubNub protocols":#pusher-pubnub
+
+h3(#mqtt). External MQTT libraries
+
+The Ably MQTT protocol adapter translates between MQ Telemetry Transport (MQTT) and Ably’s own protocol, allowing for seamless integration of any systems you may have. MQTT is a publish/subscribe, lightweight messaging protocol designed for constrained devices and low-bandwidth networks. One of the major uses of MQTT is with IoT (Internet of Things), where these principles are key to having effective communication between various devices.
+
+CAVEAT-INPUT 
+ We recommend you use MQTT to interact with Ably when:
+
+ * Using devices which do not have a device-specific Ably SDK, such as Arduino platforms, C/C++ applications, and so on
+ * If bandwidth is limited and you want to keep network traffic to a minimum
+
+Read more in our "MQTT section":/protocols/mqtt.
+
+h3(#sse-eventsource). Ably SSE and EventSource API
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime interface, or even an MQTT library, is impractical. 
+
+HTTP streaming allows for requests from clients to be held by a server, allowing it to push data to the client without further requests. This helps avoid the overhead involved in normal HTTP requests. Server-sent events (SSE) provide a thin layer on top of HTTP streaming. A common use of SSE is through the use of the EventSource API in all modern web browsers.
+
+Read more in our "SSE section":/protocols/sse.
+
+CAVEAT-INPUT 
+ Note that SSE is for subscribing to events only and you cannot publish to a channel. You will also have to handle certain operations in your client code, such as token renewal.
+
+h3(#pusher-pubnub). Pusher and PubNub protocols
+
+Ably provides the flexibility to migrate from Pusher progressively or all at once. 
+
+Our protocol adapters take care of all the background translation, you simply change your API key. You can also use a Pusher client library via this adapter to work on some platforms for which no Ably SDK is available.
+
+Take a look at our "Pusher tutorial":/tutorials/pusher-adapter and "PubNub tutorial":/tutorials/pubnub-adapter for step by step guidance.
+
+h2(#asset-tracking). Ably Asset Tracking SDKs
+
+Asset Tracking provides a way to track the location of multiple assets in realtime. An asset is any object that moves geographically, and whose position needs to be monitored. The Ably Asset Tracking solution is powered by the Ably realtime network, the Ably Asset Tracking SDKs, and the Mapbox Navigation SDK with location enhancement.
```

Comment:
I think we could at least add that the AAT SDKs are available for iOS, Android, and web clients.